docs/.vuepress/handsontable-manager/dependencies.js

```diff
@@ -19,12 +21,10 @@ const getHotUrls = (version) => {
 };
 const getFixer = () => {
   if (isBrowser) {
-    // eslint-disable-next-line no-restricted-globals
-    return [`${window.location.origin}/docs/scripts/fixer.js`, ['require', 'exports']];
+    return ['/docs/scripts/fixer.js', ['require', 'exports']];
```

Comment:
This path is also used for js-fiddle, then a full path is required.

Reply:
I checked that while testing the docs (including jsfiddle) and it seems that it worked. I will check again.

Reply:
I reverted the change be38e4b20fa3bdce3c2aa126dde84949d05dc33d.